tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -3309,8 +3309,9 @@ private static int stepsFromRoot(String origLoc) {
 
     /** Internal */
     public static void testJavaSemantics() {
-        Collator caseInsensitive = Collator.getInstance(ULocale.ROOT);
+        Collator caseInsensitive = Collator.getInstance(ULocale.ROOT); // freeze below
         caseInsensitive.setStrength(Collator.SECONDARY);
+        caseInsensitive = caseInsensitive.freeze();
```

Comment:
This collator could also be added to CollatorHelper, combining strength SECONDARY (not numeric)